spec/mailers/notify_mailer_spec.rb

```diff
@@ -36,9 +40,22 @@
     context 'when changing email' do
       it 'send confirmation to correct user' do
         response = user.send_email_changed_notification
-        expect(response.recipients[0]).to eq user.email
+        expect(response.recipients.first).to eq user.email
         expect(response.subject).to eq 'Email changed'
       end
     end
   end
+
+  describe 'unlock email' do
+    context 'when account is locked' do
+      it 'send unlock email to correct user' do
+        # response = User.send_unlock_instructions(email: user.email)
+        # expect(response.email).to eq user.email
+        
+        mail = NotifyMailer.unlock_instructions(user, :anything)
+        expect(mail.to.first).to eq user.email
```

Comment:
I'd suggest using `contain_exactly` (or another array comparison method) here rather that calling `first` on `to`. So:
```ruby
expect(mail.to).to contain_exactly(user.email)
```

Reply:
done